guacamole-ext/src/main/resources/org/apache/guacamole/protocols/ssh.json

```diff
@@ -38,6 +38,10 @@
                 {
                     "name"  : "passphrase",
                     "type"  : "PASSWORD"
+                },
+                {
+                    "name"  : "public-key"
```

Comment:
Is there a corresponding translation for the field header?

Reply:
Yeah, should probably include that. Fixed via rebase.